### PR TITLE
Fixing Umbraco Redis sample template.

### PR DESCRIPTION
**Pull request by sjwaight**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@sjwaight__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
__@sjwaight__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

File: umbraco-cms-webapp-redis-cache/azuredeploy.json [hunk 1]
```diff
@@ -1,536 +1,549 @@
...
+        "description": "Non-admin Database User. Must be Unique"
+      }
+    },
+    "dbNonAdminDatabasePassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Non-admin Database User password"
+      }
+    },
+    "storageAccountName": {
```
Comment (singhkay) [inline umbraco-cms-webapp-redis-cache/azuredeploy.json]: Consider eliminating this and generating this in the variables section. See #944

File: umbraco-cms-webapp-redis-cache/azuredeploy.parameters.json [hunk 2]
```diff
@@ -1,57 +1,63 @@
...
-         "value":"West US"
-      },
-      "databaseName":{
-         "value":"ExampleDatabase"
-      }
-   }
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "deploymentLocation": {
```
Comment (singhkay) [inline umbraco-cms-webapp-redis-cache/azuredeploy.parameters.json]: See https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#travis-ci for placeholder parameters for our CI

**Comment by sjwaight** [comment]
I ran the script using the same parameters as the above CI build and it succeeded so am unsure why it's failed here.

**Comment by singhkay** [comment]
@vivsriaus @ravbhatnagar Another test case for you guys :) The CLI returned `"error": "Deployment provisioning state was not successful\n"` Is this also fixed in the next release?

**Comment by vivsriaus** [comment]
@singhkay Can you please share the template and params, so I can test it out locally? cc: @ravbhatnagar

**Comment by singhkay** [comment]
@vivsriaus Both are in the files tab for this PR https://github.com/Azure/azure-quickstart-templates/pull/1210/files

**Comment by singhkay** [comment]
@sjwaight Tested successful deployment locally, merging

**Pull request merged**